OpenRA.Game/GameInformation.cs

```diff
@@ -35,6 +35,9 @@ public class GameInformation
 		/// <summary>Gets the game's duration, from the time the game started until the replay recording stopped.</summary>
 		public TimeSpan Duration { get { return EndTimeUtc > StartTimeUtc ? EndTimeUtc - StartTimeUtc : TimeSpan.Zero; } }
 		public IList<Player> Players { get; private set; }
+
+		public List<int> DisabledSpawnPoints;
+
```

Comment:
Style nit: Can we remove the empty lines above and below the new list again to be consistent with the other properties?